server/src/index.ts

```
@@ -487,22 +413,16 @@ function blockSocket(socket: universal.GameSocket) {
-app.post(
-  "/authenticate",
-  jsonParser,
-  async (request: Request, response: Response) => {
-    const username = request.body["username"];
-    const password = request.body["password"];
-    const socketID = request.body["socketID"];
-    const result = await authenticate(username, password, socketID);
-    response.json({ success: result });
-  }
-);
+require("fs")
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)